scapy/route.py

```diff
@@ -21,8 +22,11 @@
 
 class Route:
     def __init__(self):
-        self.resync()
-        self.cache = {}
+        if isinstance(conf.route, list):
```

Comment:
Why "isinstance" check? Could you at least add a comment? (Preferably change the condition to "is not None", "!= """ or similar.